platform/platform-configuration-migration/src/test/java/org/codice/ddf/configuration/admin/ConfigurationAdminMigrationTest.java

```diff
@@ -622,13 +580,12 @@ void moveFile(Path source, Path destination) throws IOException {
     @Test
     public void testNotifyWithValidFile() throws Exception {
```

Comment:
Is this test really testing the following:
A file fails import, it is fixed, it is re-imported and is processed?  If so, the name of the method should be changed to something like testNotifyWhenFileIsReprocessed().  Thoughts?